src/lib/y2storage/boot_requirements_strategies/prep.rb

```diff
@@ -55,6 +64,21 @@ def warnings
 
     protected
 
+      # PReP partition is needed, so return any warning related to it.
+      def prep_warnings
+        res = []
+
+        if missing_partition_for?(prep_volume)
+          res << SetupError.new(missing_volume: prep_volume)
+        elsif prep_too_big?
+          msg = _("PReP partition is too big. " \
+           "Some firmwares can refuse to load it and prevent booting.")
+          res << SetupError.new(message: msg)
```

Comment:
Well, we should tell the users which is the max size unless we want them to start a trial and error process. Something like "PReP partition is too big. Some firmwares are unable to load PReP partitions bigger than %s. If that happens the system will not boot." Just an idea. Any sentence mentioning the limit should make it.

Reply:
Moreover, I just saw down there that we check for any PReP partition being too big. Even those that are not located in the same disk than "/". In that case maybe we should include the name of the partition in the message or something. Maybe I want to install in sda and I create a proper "/" and PReP there. I would like to know if the warning is actually referring to sdb. In that case, I don't care.